### PR TITLE
PS 8 compatibility

### PR DESCRIPTION
**Pull request by Afayadas**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add compatibility with prestashop 8
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Task #37392
| How to test?  | Install and use module.

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 name: Coding Standart
-on: [pull_request]
+on: [push]
```
Comment (clotairer) [inline .github/workflows/php.yml:2]: re-turn to "pull_request" Tx

File: stripe_official.php [hunk 2]
```diff
@@ -1663,7 +1664,7 @@ public function hookActionOrderStatusUpdate($params)
     /**
      * Load JS on the front office order page
      */
-    public function hookHeader()
+    public function hookDisplayHeader()
```
Comment (clotairer) [inline stripe_official.php:1667]: are you sure for 1.6.0, you don't need an upgrade ?

**Review by clotairer — CHANGES_REQUESTED** [review]

**Comment by clotairer** [comment]
```
  ------ -------------------------------------------------------------------- 
  Line   modules/stripe_official/classes/actions/ValidationOrderActions.php  
 ------ -------------------------------------------------------------------- 
  658    Call to an undefined static method OrderPayment::getByOrderId().    
 ------ -------------------------------------------------------------------- 
```

File: stripe_official.php (hunk 2 above)
Comment (Afayadas) [reply stripe_official.php:1667]: hook displayHeader is full compatible with ps 1.6

**Review by clotairer — APPROVED** [review]

**Pull request merged**